packages/api/src/controllers/experiment/attestation.test.ts

```diff
@@ -126,6 +126,32 @@ describe("Attestation API", () => {
   });
 
   describe("POST /", () => {
+    it("should support Flow Wallet signatures", async () => {
```

Comment:
Can we add a test for it to reject invalid signatures? Maybe cross-user

Reply:
Updated in e25a2ac02b4c98893638298020875cf891cb6ce3